ADAL/src/telemetry/ADTelemetryDefaultEvent.m

```diff
@@ -135,22 +134,19 @@ - (NSString*)getStringFromDate:(NSDate*)date
 
 - (void)addAggregatedPropertiesToDictionary:(NSMutableDictionary*)eventToBeDispatched
 {
-    for (int i=0; i<[self getDefaultPropertyCount]; i++)
-    {
-        NSString* propertyName = [(ADTelemetryProperty*)_propertyMap[i] name];
-        NSString* propertyValue = [(ADTelemetryProperty*)_propertyMap[i] value];
-        [eventToBeDispatched setObject:propertyValue forKey:propertyName];
-    }
+    [eventToBeDispatched addEntriesFromDictionary:[self defaultParameters]];
+    SET_IF_NOT_NIL(eventToBeDispatched, AD_TELEMETRY_REQUEST_ID, [_propertyMap objectForKey:AD_TELEMETRY_REQUEST_ID]);
```

Comment:
currently the aggregation feature by design is to aggregate all relevant events related to a particular acquireToken call into one single event.